themes/src/main/resources/theme/base/admin/resources/partials/client-detail.html

```diff
@@ -37,6 +37,13 @@
                 </div>
                 <kc-tooltip>{{:: 'client.enabled.tooltip' | translate}}</kc-tooltip>
             </div>
+            <div class="form-group clearfix block" data-ng-show="serverInfo.featureEnabled('ACCOUNT2')">
```

Comment:
Should we make this invisible for bearer only clients and those with no base url. The users may get confused if they set alwaysDisplayInConsole and then the client isn't displayed.

Reply:
@douglaspalmer @abstractj Yes, bearer only clients need to be handled properly in the UI.  I think it will need a trigger so that it sets alwaysDisplayInConsole to false and disables the selection whenever bearer only is set.

I'm not sure about the "no base url" thing.  Are we unable to show in account console if there is no base url?